src/rprblender/utils/helper_lib.py

```diff
@@ -67,8 +67,16 @@ def init():
                     f"{os.environ.get('LD_LIBRARY_PATH', '')}"
 
     else:
+        root_dir = package_root_dir()
         lib_dir = package_root_dir()
 
+    if IS_WIN and IS_OPENVDB_SUPPORT:
+        if IS_DEBUG_MODE:
+            ctypes.CDLL(str(root_dir /
+                'RadeonProRenderSharedComponents/OpenVdb/Windows/bin/Half-2_3.dll'))
```

Comment:
we include the dll in our shipment right?

Reply:
yes, they are in zip installer